pkgs/by-name/ni/niri/package.nix

```diff
@@ -82,6 +83,10 @@ rustPlatform.buildRustPackage (finalAttrs: {
   buildNoDefaultFeatures = true;
 
   postInstall =
+    let
+      niri = "${stdenv.hostPlatform.emulator buildPackages} ${placeholder "out"}/bin/niri";
```

Comment:
usage of emulator is discouraged, see https://github.com/NixOS/nixpkgs/pull/422712